### PR TITLE
EQL: fix spec test exampleWithTwoStagesAndKeyFromTheDocs

### LINKED ISSUE #67102: EQL: [Tests] exampleWithTwoStagesAndKeyFromTheDocs sequence test fails
Special test with keys fails:

```
  2> java.lang.AssertionError: Line 251:different sequences matched  expected:<2> but was:<0>
        at __randomizedtesting.SeedInfo.seed([13E472B2782A612:896A78F1897ECBEA]:0)
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:645)
        at org.elasticsearch.xpack.eql.execution.assembler.SequenceSpecTests.checkResults(SequenceSpecTests.java:233)
        at org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:128)
```

This was hidden since the SequenceSpecTests were not correctly asserted before.

The specific test has been commented out for `master`, `7.x`, `7.11` and `7.10` branches.

### PR DESCRIPTION
**Pull request by luigidellaquila**
Fixes #67102

### CODE REVIEW

File: x-pack/plugin/eql/src/test/resources/sequences.series-spec [hunk 1]
```diff
@@ -249,12 +249,12 @@ A4 B5 C7 D8
 // https://eql.readthedocs.io/en/0.8/query-guide/implementation.html#sequences
 
 // AwaitsFix: https://github.com/elastic/elasticsearch/issues/67102
```
Comment (astefan) [inline x-pack/plugin/eql/src/test/resources/sequences.series-spec:251]: This line should be removed as well.

**Review by astefan — APPROVED** [review]
LGTM.  Left one minor comment.

**Comment by luigidellaquila** [comment]
@elasticmachine  run elasticsearch-ci/bwc

**Comment by luigidellaquila** [comment]
@elasticmachine update branch

**Comment by luigidellaquila** [comment]
@elasticmachine run elasticsearch-ci/bwc

**Comment by luigidellaquila** [comment]
@elasticmachine update branch

**Pull request merged**